### ISSUE TITLE
Pandas deprecated functions should be reviewed

### ISSUE THREAD

**Issue by LeninGF** [dashaub/espandas#6]
Greetings, I was testing your software. Congratulations it looks very functional, However there are problems with the most recent versions of pandas since reindex_axis is no longer avaible in pandas 1.1.3 I will try to make a correction if I get sometime. In the mean time I report this bug and people that may use this code should consider runing a specific version of pandas.

**Comment by dashaub** [comment]
@LeninGF 
Yes, the pinned pandas version is very old. But so is the ElasticSearch version, and I'd be more worried about that.

Which version of ES are you using? Does it work? I haven't tested this on anything newer.
